tests/unit/test_connection_cmds.py

```diff
@@ -973,7 +974,7 @@ def GET_TEST_PATH_STR(filename):  # pylint: disable=invalid-name
       'rc': 1,
       'test': 'innows',
       'file': {'before': 'none', 'after': 'none'}},
-     None, RUN],
+     None, OK],
```

Comment:
In this PR you changed it to OK, in the part 2 PR you changed it back to RUN. I suggest to remove this change here.

Reply:
Marked OK here and now OK in next pr